src/project/qml/MuseScore/Project/internal/Migration/MigrationContentForPre362.qml

```diff
@@ -70,71 +68,51 @@ Item {
         }
     }
 
-    Column {
-        id: mainContent
-        anchors.top: parent.top
-        anchors.left: parent.left
-        anchors.right: parent.right
-        anchors.margins: 16
-        height: childrenRect.height
-
-        property NavigationPanel navigationPanel: NavigationPanel {
-            name: "MigrationContentPanel"
-            section: root.navigationSection
-            accessible.role: MUAccessible.Dialog
-            order: 2
-        }
+    ColumnLayout {
+        id: header
+        spacing: 8
 
         StyledTextLabel {
             id: headerTitle
-            anchors.left: parent.left
-            anchors.right: parent.right
-            height: 32
-
             font: ui.theme.tabBoldFont
-
-            horizontalAlignment: Qt.AlignLeft
-            verticalAlignment: Qt.AlignVCenter
-
             text: qsTrc("project/migration", "This file was last saved in MuseScore %1").arg(root.appVersion)
         }
 
         StyledTextLabel {
             id: headerSubtitle
-            anchors.left: parent.left
-            anchors.right: parent.right
-            height: 24
-
-            horizontalAlignment: Qt.AlignLeft
-            verticalAlignment: Qt.AlignVCenter
-
             text: qsTrc("project/migration", "Select the engraving improvements you would like to apply to your score")
         }
+    }
 
-        Item {
-            id: imageItem
-            anchors.left: parent.left
-            anchors.right: parent.right
-            height: 264
-
-            Image {
-                anchors.left: parent.left
-                anchors.right: parent.right
-                anchors.verticalCenter: parent.verticalCenter
-                height: 200
-                fillMode: Image.PreserveAspectFit
-                source: "migration.png"
-            }
+    Rectangle {
+        id: imageRect
+        color: "#F9F9F9"
+        height: 178
+        radius: 10
+
+        border.width: 1
+        border.color: ui.theme.strokeColor
+
+        Layout.fillWidth: true
+
+        Image {
+            source: "migration.png"
+            height: 143
```

Comment:
```suggestion
            height: 158
```
(The image has 2x resolution, and you cropped 20 pixels from each side, which corresponds to 10 pixels at each side in QML; so it should be 178 - 2 * 10.)

Reply:
So QML height should be half the pixel height of the image?

(I think I took the previous value from Figma)

Reply:
I think so, at least that's how it's right now on Figma, and I think that's likely to result in the best quality (1:1 on high res displays, 2:1 on normal displays, rather than something less round)